### DISCUSSION TITLE
Error Occurred While Running Visualizations

### DISCUSSION THREAD

**Discussion by apixler77** [utmstack/UTMStack#746] [Q&A]
Keep getting the above error when opening different dashboards. Running version 10.5.5 on Ubuntu 22.04 inside of VMWare Workstation.

**Comment by c3s4rfred** [comment]
Hi, @apixler77, can you provide some images and logs for us please. Best regards

**Reply by apixler77** [reply]
What logs would you like?

**Reply by c3s4rfred** [reply]
We can start with some images of the error that you are saying, and logs from the "backend" docker container. You can use 'docker ps' and 'docker logs containerId' commands from the UTMStack master server.

Best regards

**Reply by apixler77** [reply]
I am attaching the docker ps, but when running the docker logs containerld command I get back there is no such file.
[docker ps.txt](https://github.com/user-attachments/files/16383751/docker.ps.txt)

**Reply by c3s4rfred** [reply]
docker ps command is just to copy the containerId value for ghcr.io/utmstack/utmstack/backend:v10 image, in the file that you attached the containerId = 6c585e337f6d, then use docker logs 6c585e337f6d to get the backend logs.

Best regards

**Reply by apixler77** [reply]
That log is huge, do you want just the errors that it is showing?

**Reply by apixler77** [reply]
Okay, so I got a bunch of the errors from that log file as well as a screenshot inside of UTMStack.
![UTMStack Errors](https://github.com/user-attachments/assets/2bacbac9-0eed-4e04-a0f7-af30355ccac4)
[docker logs 6c585e337f6d.txt](https://github.com/user-attachments/files/16395733/docker.logs.6c585e337f6d.txt)

**Reply by c3s4rfred** [reply]
Hi, @apixler77, please try to restart the machine and tell us if the issue persists, the problem is related to the opensearch instance, for some reason is not running well.

Best regards

**Reply by apixler77** [reply]
Just rebooted machine and got the same error. I have attached it. Also not getting Meraki logs or Sentinel One.
![UTMStack Error Again](https://github.com/user-attachments/assets/b143ad7a-93a8-4449-96ce-e7def4692f75)

**Reply by c3s4rfred** [reply]
Hi, @apixler77, I can see in the image that only one visualization is having issues, please can you click on this kind of error and send an image to us -> 
![image](https://github.com/user-attachments/assets/78215686-229b-4b0f-a751-d45a90395bcd)
Also, can you try first to reinstall and, if the issue persists, try to install from scratch in other vm using our guides -> https://docs.utmstack.com/, please, give special atention to the system requirements. Let us know the results.

**Reply by apixler77** [reply]
I am working on the reinstall this week. I will let you know what happens.

**Reply by apixler77** [reply]
I have attempted to install UTMStack 3 different times now. The installer stops at "backend start failure". Cannot get the server to install. Ubuntu 22.04 on Virtual Box.

**Reply by c3s4rfred** [reply]
Hi @apixler77 that error happens when the backend can't communicate with opensearch, postgres or environment variables are not set (this can't be possible since the installer creates de variables). During installation we created a virtual network on linux system, that can cause the backend to fail too. So, we recommend a fresh system installation, new VM, and then try again.

Best regards

**Reply by apixler77** [reply]
Ended up downloading and installing from ISO. This is done.
